### ISSUE TITLE
Access is denied

### ISSUE THREAD

**Issue by aponteju** [unosquare/passcore#216]
After hitting "Change password", I am getting a message at the bottom of the screen that says "Access is denied." There is no further information. I have tried with and without LDAP credentials. I have set the default domain name. 

Any thoughts?

**Comment by jaldinger** [comment]
Same problem here. Did you discover any new information?

**Comment by jaldinger** [comment]
In the meantime, I went back to v3.1.0 and it works fine there...

**Comment by geoperez** [comment]
Question. Did you use the same settings file between versions?

**Comment by jaldinger** [comment]
I tried with the old and the new. I also compared them and found the only section to be different the "LdapPasswordChangeOptions" section. Didn't seem to have any effect on the error message. I enabled the StdOut log but didn't find any helpful information in the log also.

**Comment by aponteju** [comment]
@geoperez No, I downloaded a fresh copy of the latest version and set up the new Appsettings file. I also set this up on a brand new domain joined server to eliminate possible errors from the local machine. I'm wondering if the issue is with Active Directory or the application?

**Comment by geoperez** [comment]
I'm going to check if I can increase the logging details to ease troubleshooting.

**Comment by greecemunky** [comment]
I'm also experiencing the same issue, and have also reverted to PassCore v3.1.0.  I didn't reuse my older config files - started fresh, updating only the necessary fields.

My environment:

- Windows Server 2008 R2
- IIS 7
- Windows Management Framework 5.1
- .NET Core 2.1 Runtime & Hosting Bundle for Windows (v2.1.1)

POST to https://my.passcore.instance/api/password returns error "400 Bad Request".

Request (hostname, username and passwords have been manually obscured):

```
POST /api/password HTTP/1.1
Host: my.passcore.instance
Connection: keep-alive
Content-Length: 102
Accept: application/json, text/plain, */*
Origin: https://my.passcore.instance
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.92 Safari/537.36
Content-Type: application/json
Referer: https://my.passcore.instance/
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.9
Cookie: __utma=68798416.2097907996.1538369690.1538369690.1538369690.1; __utmc=68798416; __utmz=68798416.1538369690.1.1.utmcsr=(direct)|utmccn=(direct)|utmcmd=(none)

{"Username":"testuser","CurrentPassword":"myoldpass","NewPassword":"mynewpw","NewPasswordVerify":"mynewpw"}
```

Response:

```
HTTP/1.1 400 Bad Request
Transfer-Encoding: chunked
Content-Type: application/json; charset=utf-8
Server: Kestrel
X-Powered-By: ASP.NET
Date: Wed, 03 Oct 2018 04:40:22 GMT

{"hasErrors":true,"errors":[{"errorCode":0,"fieldName":null,"message":"Access is denied.\r\n"}],"payload":null}
```

Request POST data looks identical to that submitted by PassCore v3.1.0.

Response in v3.1.0 (successful):

```{"hasErrors":false,"errors":[],"payload":null}```

**Comment by greecemunky** [comment]
I created another site and Application Pool to test v3.4.2.

Changing the Application Pool's identity (in IIS 7, select App Pool > Advanced Settings > Process Model > Identity) from ApplicationPoolIdentity to LocalSystem appears to work (mine runs on a DC, if that matters).

Are there's any implications running it as LocalSystem?

**Comment by aponteju** [comment]
I tried running as LocalSystem but that didn't work. Still getting the same error. I don't think it's best practice to run IIS on a DC, so I'd rather not go that route.

Let me see if I can try 3.1.0

**Comment by aponteju** [comment]
I tested 3.1.0. With applying the same settings as 3.4.0, I got "could not access restricted groups" or something to that effect. When I set the RestrictedADGroups to false, the error went away and my password change succeeded.

**Comment by geoperez** [comment]
Please try the new version:

https://github.com/unosquare/passcore/releases/download/3.6.0/PassCore36.zip
